### PR TITLE
Add kotlin code for the chapter of graph

### PR DESCRIPTION
**Pull request by curtishd**
If this pull request (PR) pertains to **Chinese-to-English translation**, please confirm that you have read the contribution guidelines and complete the checklist below:

- [ ] This PR represents the translation of a single, complete document, or contains only bug fixes.
- [ ] The translation accurately conveys the original meaning and intent of the Chinese version. If deviations exist, I have provided explanatory comments to clarify the reasons.

If this pull request (PR) is associated with **coding or code transpilation**, please attach the relevant console outputs to the PR and complete the following checklist:

- [x] I have thoroughly reviewed the code, focusing on its formatting, comments, indentation, and file headers.
- [x] I have confirmed that the code execution outputs are consistent with those produced by the reference code (Python or Java).
- [x] The code is designed to be compatible on standard operating systems, including Windows, macOS, and Ubuntu.
屏幕截图 2024-03-03 154343
屏幕截图 2024-03-03 154350
屏幕截图 2024-03-03 154404
屏幕截图 2024-03-03 154414
屏幕截图 2024-03-03 154459
屏幕截图 2024-03-03 154509

### CODE REVIEW

**Comment by krahets** [comment]
Please set the PR's title as "Add kotlin code for the chapter of graph". I've modified the title of this PR

File: codes/kotlin/chapter_graph/graph_adjacency_matrix.kt [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    }
+
+    /* 获取顶点数量 */
+    fun size():Int{
+        return vertices.size
+    }
+
+    /* 添加顶点 */
+    fun addVertex(value:Int){
+        val n=size()
```
Comment (krahets) [inline codes/kotlin/chapter_graph/graph_adjacency_matrix.kt:36]: Is the code unformatted?

**Comment by curtishd** [comment]
Hmm, Some imperfections are not easy to spot.

**Comment by krahets** [comment]
> Hmm, Some imperfections are not easy to spot.

Indeed

**Review by krahets — APPROVED** [review]
Great work! Thanks!

**Comment by curtishd** [comment]
My pleasure

**Pull request merged**